contracts/pallet-ibc/src/ics20/mod.rs

```diff
@@ -42,6 +42,26 @@ use ibc_primitives::{CallbackWeight, HandlerMessage, IbcHandler};
 use sp_core::crypto::AccountId32;
 use sp_std::marker::PhantomData;
 
+pub type Ics20TransferMsg = ibc::applications::transfer::msgs::transfer::MsgTransfer<
+	ibc::applications::transfer::Coin<ibc::applications::transfer::PrefixedDenom>,
+>;
+
+#[derive(Debug, Clone, Copy)]
+pub enum FlowType {
```

Comment:
can you put some comments on what each of these mean?

Reply:
It is 100% same as Extrinsic names which lead to transfers. Assuming that later the date some protocols will transfer too(they could be different settings). So it is not in and out. Imho naming according extinsics origin is good enough docs. Also enum in reference code of cosmos has not comments on it. So there are 2 reasons not to have doc comments.

Reply:
There could be 3rd way when message is just sent from liquidation engine, it does not call extinsic. so it would not be subject to rate limiter or different setting. so just retaining room for `InternalTransfer` (while on delivery side I do not care so).